components/CreateHabitListForm.js

```diff
@@ -63,11 +74,9 @@ export default function CreateHabitListForm() {
           { errorMessage && <p className="text-red-500 text-xs italic mt-2"> { errorMessage } </p> }
         </div>
         <div className="flex items-center justify-between">
-          <button className="bg-blue-500 hover:bg-blue-700 text-white font-bold text-xl w-96 h-12 py-1 rounded focus:outline-none focus:shadow-outline" type="submit">
-            create new list
-          </button>
+          <FormButton text="Create new list" loading={showLoader} disabled={showLoader} />
```

Comment:
do we need two separate props for FormButton here? I imagine that if its in loading state, it should always be disabled--so we can remove the `disabled` prop here and then set `disabled={loading} ` in FormButton